docs/faq.md

```diff
@@ -77,7 +77,7 @@ vds.virtualize.to_icechunk(icechunkstore)
 
 ### I already have some data in Zarr, do I have to resave it?
 
-No! VirtualiZarr can (well, [soon will be able to](https://github.com/zarr-developers/VirtualiZarr/issues/262)) create virtual references pointing to existing Zarr stores in the same way as for other file formats.
+No! VirtualiZarr can create virtual references pointing to existing Zarr stores in the same way as for other file formats. Note: Currently only reading Zarr V3 is supported.
```

Comment:
Do we have an issue to track learning to read zarr v2?

Reply:
https://github.com/zarr-developers/VirtualiZarr/issues/565